pkg/services/accesscontrol/acimpl/service.go

```diff
@@ -59,16 +68,26 @@ func (s *Service) GetUsageStats(_ context.Context) map[string]interface{} {
 }
 
 var actionsToFetch = append(
-	ossaccesscontrol.TeamAdminActions, append(ossaccesscontrol.DashboardAdminActions, ossaccesscontrol.FolderAdminActions...)...,
+	ossaccesscontrol.TeamAdminActions,
+	append(ossaccesscontrol.ServiceAccountAdminActions,
```

Comment:
nit: is this change related?

Reply:
No, will revert this change and create issue for this